src/gui/FileServerList.C

```diff
@@ -1511,6 +1515,16 @@ FileServerList::OpenAndGetMetaData(const QualifiedFilename &filename,
                 openFileTimeState = timeState;
                 fileAction = action;
                 Select(ID_fileAction, (void *)&fileAction);
+                // add to appliedFilesList if not already there
+                bool found = false;
+                for (size_t i = 0; i < appliedFileList.size() && !found; ++i)
+                    found = appliedFileList[i] == openFile;
```

Comment:
Would this code be better in a separate method? e.g, bool isInFileList(file)
Not sure if it could be needed by other code in the future or if the logic would need to be updated. 
Just a thought...I would defer to your decision.

Reply:
Not sure we want to make it accessible outside the class. I suppose if we find it's needed elsewhere in the future, it could be abstracted out.